charts/jxboot-helmfile-resources/templates/jx-gcactivities-cronjob.yaml

```diff
@@ -29,13 +29,14 @@ spec:
                 - gitops
                 - gc
                 - activities
+                - "--all-namespaces"
```

Comment:
This one requires https://github.com/jenkins-x-plugins/jx-gitops/pull/835 to be merged first.